ts/lib220.ts

```diff
@@ -1,5 +1,52 @@
 var runner: any = undefined;
 
+var ImageData: any = ImageData;
+
+if (typeof ImageData === 'undefined') {
+  ImageData = class {
+    width: number = 1;
+    height: number = 1;
+    data: Uint8ClampedArray = new Uint8ClampedArray(4);
+    constructor(width: number, height: number);
+    constructor(array: Uint8ClampedArray, width: number, height: number);
+    constructor(widthOrArray: number | Uint8ClampedArray, widthOrHeight: number, height?: number) {
```

Comment:
No variants. Take exactly 2 arguments, one for width, one for height. Initialize all pixels to 0.

Reply:
Yeah. Have exactly one constructor. You can have helper functions if you want.

Reply:
Changes pushed.
FYI: ImageData is not going to be used by students, they're only going to be using functions that are exported. I made it take 3 or 2 arguments to mimic exactly how ImageData class is for the browser.